newsletter-may-2023.rst

```diff
@@ -0,0 +1,79 @@
+.. post:: May 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
```

Comment:
This is a more general note but I assume that a lot of newsletters have the exact same meta description. Duplicate meta descriptions are mostly ignored by search engines. We should either customize it a bit -- even if you just add the month and year -- or just remove it.

Reply:
Removed!